core/src/main/java/org/apache/gravitino/EntityStore.java

```diff
@@ -194,4 +195,8 @@ <R, E extends Exception> R executeInTransaction(Executable<R, E> executable)
   default SupportsTagOperations tagOperations() {
     throw new UnsupportedOperationException("tag operations are not supported");
   }
+
+  default SupportsRelationOperations relationOperations() {
```

Comment:
This method is more abstract and thus more preferable than `SupportsTagOperations` if it can fulfill all functions provided by `SupportsTagOperations`.